/**
 * Error -> happens - can't recover
 * Exception -> handled programatically
 */

/*
//syntax related

console.log("hello   // SyntaxError -> not allow your program to run

//runtime exception

let x = 5;

console.log(x.toUppercase());

//logical 

let num = 5;
for(let i=-3; i<5; i++){
    console.log(num/i)         //--> we can't divide any number by zero
}
*/


